src/nekontrol/console.py

```diff
@@ -0,0 +1,17 @@
+from rich.console import Console
+
+_console = None
```

Comment:
rich documentation suggested having a shared console so it isn't initialized multiple times by accident. not sure if i agree but here it is for your consideration.

Reply:
It would be nice to have a shared console, maybe it should be passed to every function that requires some kind of console interaction since it would be easier to trace IO effects.

Reply:
true. do you know if click has something like pytest's fixtures?